app/models/tenant.rb

```diff
@@ -266,4 +270,17 @@ def validate_only_one_root
       errors.add(:parent, "required") if root && root != self
     end
   end
+
+  def create_tenant_group
+    if default_miq_group_id.nil? && default_miq_group.nil?
```

Comment:
Not sure why you need the second half of the condition here.  Doesn't the first imply the second, and if not, wouldn't the second obviate the need for the first?